settings.gradle

```diff
@@ -12,11 +12,14 @@ include 'backendImpl'
 include 'frontendActionIds'
 include 'frontendActions'
 include 'frontendDataKeys'
+include 'frontendFile'
 include 'frontendGraphApi'
 include 'frontendGraphImpl'
+include 'frontendIcons'
 include 'frontendUiApi'
 include 'frontendUiTableImpl'
 include 'frontendUiRootImpl'
+include 'frontendVfsUtils'
```

Comment:
Nit: I'd keep topological order instead of alphabetical order here, so both `frontendIcons` and `frontendVfsUtils` should be higher up

Reply:
...and actually also `frontendDataKeys` should be before `frontendActions`, my bad